src/Sylius/Bundle/PayumBundle/DependencyInjection/SyliusPayumExtension.php

```diff
@@ -44,4 +45,23 @@ public function load(array $config, ContainerBuilder $container)
         $container->setParameter('payum.template.layout', $config['template']['layout']);
         $container->setParameter('payum.template.obtain_credit_card', $config['template']['obtain_credit_card']);
     }
+
+
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        if (!$container->hasExtension('sylius_payment')) {
+            return;
+        }
+
+        $config = $container->getExtensionConfig('payum');
```

Comment:
this is not really accurate since some gateways could be added via container tag, or even stored in database.